src/components/pages/NewRisk/NewRisk.js

```diff
@@ -31,22 +31,22 @@ const mapContainerStyle = {
   minHeight: "250px",
   maxheight: "80vh",
   margin: "0 auto",
-}
+};
 
 // ******* projectID will need to be retrieved
 const projectId = "5f52a6cc0c5677512c956ded";
 
 // ******* mapCentre will need to be retrieved
 let mapCentre = {
   lat: 52.475,
-  lng: -1.900,
+  lng: -1.9,
 };
```

Comment:
Remove this variable, instead pass this object into the useState on line 63

Reply:
variable removed